src/assets/styles/fonts/titillium-web.scss

```diff
@@ -2,8 +2,8 @@
   font-display: swap;
   font-family: 'Titillium Web';
   font-style: normal;
-  font-weight: 900;
-  src: local('Titillium Web Black'), local('TitilliumWeb-Black'), url('../../fonts/TitilliumWeb/TitilliumWeb-Black.ttf') format('truetype');
+  font-weight: 500;
```

Comment:
Is this for "normal" font-weight? Normal is 400.

Reply:
@undsoft 
<img width="1300" alt="Screenshot 2024-09-05 at 18 34 38" src="https://github.com/user-attachments/assets/9a1dd79d-1fab-463e-80d8-3b426f1dda32">

Reply:
I think we need to remove `font-style: normal;` in this case